### PR TITLE
Proj0800 & Proj0801: Use Central Package Version Management

### PR DESCRIPTION
**Pull request by Corniel**
`<ManagePackageVersionsCentrally>` should be configured, and when enabled, the `Directory.Packages.props` should be resolvable.

#143

### CODE REVIEW

**Comment by Corniel** [comment]
Documentation can be found here: https://github.com/dotnet-project-file-analyzers/dotnet-project-file-analyzers.github.io/pull/21

File: src/DotNetProjectFile.Analyzers/README.md [hunk 1]
```diff
@@ -46,6 +46,8 @@ The package contains analyzers that analyze .NET project files.
 * [**Proj0242** Generate NuGet packages conditionally](https://dotnet-project-file-analyzers.github.io/rules/Proj0242.html)
 * [**Proj0400** Define the project publishability explicitly](https://dotnet-project-file-analyzers.github.io/rules/Proj0400.html)
 * [**Proj0600** Avoid generating packages on build if not packable](https://dotnet-project-file-analyzers.github.io/rules/Proj0600.html)
+* [**Proj0800** Configure Central Package Version Management](https://dotnet-project-file-analyzers.github.io/rules/Proj0800.html)
+* [**Proj0800** Include 'Directory.Packages.props'](https://dotnet-project-file-analyzers.github.io/rules/Proj0800.html)
```
Comment (LauraXiulan) [inline src/DotNetProjectFile.Analyzers/README.md:50]: 0801

File: specs/DotNetProjectFile.Analyzers.Specs/Rules/MS_Build/Include_Directory_Packages_props.cs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        => new IncludeDirectoryPackagesProps()
+        .ForProject("NoDirectoryPackagesProps.cs")
+        .HasIssue(new Issue("Proj0801", "The file 'Directory.Packages.props' could not be located.")
+        .WithSpan(00, 00, 00, 32));
+}
+
+public class Guards
+{
+    [Test]
+    public void Projects_with_CPMV_file()
```
Comment (LauraXiulan) [inline specs/DotNetProjectFile.Analyzers.Specs/Rules/MS_Build/Include_Directory_Packages_props.cs:16]: CPVM

File: dictionary.dic [hunk 3]
```diff
@@ -1,4 +1,6 @@
-﻿fallback
+﻿CPVM
```
Comment (Sjaaky) [inline dictionary.dic:1]: CPVM?
It is Central Package Management (CPM).

**Review by Sjaaky — COMMENTED** [review]
Central Package Management (CPM)

File: src/DotNetProjectFile.Analyzers/MsBuild/ProjectFileTypes.cs [hunk 4]
```diff
@@ -4,12 +4,18 @@ public static class ProjectFileTypes
 {
     public static readonly IReadOnlyCollection<ProjectFileType> ProjectFile = [ProjectFileType.ProjectFile];
 
-    public static readonly IReadOnlyCollection<ProjectFileType> ProjectFileAndProps =
+    public static readonly IReadOnlyCollection<ProjectFileType> ProjectFile_Props =
```
Comment (CptWesley) [inline src/DotNetProjectFile.Analyzers/MsBuild/ProjectFileTypes.cs:7]: `Directory.Build.props` and `Directory.Packages.props` are also very much just `.props` file that can be used for defining anything, so shouldn't we include them here as well?

Comment (Corniel) [reply src/DotNetProjectFile.Analyzers/MsBuild/ProjectFileTypes.cs:7]: @CptWesley For those rules (the default) `ProjectFileTYpes.All` exits. For some cases, those special cases are excluded. (The requirement to be an additional file for instance).

**Comment by Corniel** [comment]
@Sjaaky I think I addressed all CPVM => CPM remarks.

**Review by CptWesley — APPROVED** [review]

**Pull request merged**